database/mock_data/PopulateDB_Part2.sql

```diff
@@ -29,4 +29,7 @@ update membership set membership_admin_id = 1 where membership_id = 2; -- this w
 update member m
 left join membership ms on m.membership_id = ms.membership_id
 set membership_admin_id = m.member_id
-where m.member_type_id = 2;
\ No newline at end of file
+where m.member_type_id = 2;
+
+CALL sp_event_job_generation('2022-04-01T08:00:00', '2022-04-01T16:00:00', 1, '2022 Last Race', 'test empty event', @ignore);
```

Comment:
Please put this up top with the rest of the calls to `sp_event_job_generation`

Reply:
This was intentional (per Jacob's idea) to create an event with jobs which aren't assigned to members yet

Reply:
No, I was just trying to keep it organized. If there's a reason for it then this is fine 👍